docs/reference/field-types/color.md

```diff
@@ -17,6 +31,7 @@ A `color` field provides a colorpicker interface to the editor for choosing/past
 |---|---|---|---|
 |name | `string` | | Sets the name of the field in the database |
 |label | `string` | | Sets the label of the field that the user sees |
+|spectrumOptions | `object` | | Provide custom configuration to the colorpicker interface |
```

Comment:
There is a default to include: 
```
{
          showButtons: true,
          showAlpha: true,
          preferredFormat: 'hex',
          showInput: true,
          allowEmpty: true
        };
```

I don't know if that can fit in a table cell, but it could be added above and then referenced with a "see above" here.